deploy/rbac.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 kind: Role
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: local-storage-operator
 rules:
```

Comment:
While we are doing this - should we also update the CRDs?

Reply:
CRDs are already updated to v1.